atomone-1/README.md

```diff
@@ -0,0 +1,80 @@
+# 🔗 `atomone-1`
+
+![chain-id](https://img.shields.io/badge/chain%20id-atomone--1-blue?style=for-the-badge)
+![version: v1.0.0](https://img.shields.io/badge/version-v1.0.0-green?style=for-the-badge)
+<!-- ![genesis-time](https://img.shields.io/badge/%E2%8F%B0%20genesis%20time-2024--02--27T13%3A00%3A00Z-red?style=for-the-badge) -->
+
+
+## Register in the Genesis
+
+To register your validator node in the `genesis.json` you just need to provide a signed `gentx` with an initial delegation of `1000000uatone`.
+
+```sh
+# Init node
+atomeoned init your-moniker --chain-id atomone-1
+
+# Create keys, be careful with the mnemonic 👀
+atomeoned keys add your-key-name
+
+# Set account necessary balance
+atomeoned add-genesis-account your-key-name 10000000uatone
+```
+*NOTE*: if you are submitting a gentx for a new account, (meaning you had to run the command above because the account did not have balance) be aware that you will be granted some funds (including the 1000000uatone required as min self-delegation), enough to pay for fees. The amount above is indicative of the actual amount of funds you will be granted, but the final amount might change.
+
+Then create your own genesis transaction (`gentx`). You will have to choose the following parameters for your validator: `commission-rate`, `commission-max-rate`, `commission-max-change-rate` all set to 0, `min-self-delegation` (>=1), `website` (optional), `details` (optional), `identity` ([keybase](https://keybase.io) key hash, used to get validator logos in block explorers - optional), `security-contact` (email - optional).
+
+The `commission-rate`, `commission-max-rate`, `commission-max-change-rate` are recommended to be set to 0  since rewards and inflations are both set to 0.
+
+```sh
+# Create the gentx
+atomeoned gentx your-key-name 1000000uatone \
+  --node-id $(atomeoned tendermint show-node-id) \
+  --chain-id atomone-1 \
+  --commission-rate 0.05 \
```

Comment:
I've put 5% by default, what does the community think of this ?

Reply:
5% is the min commission defined in the genesis, so any number below that will cause a rejection of the gentx.

Reply:
Since this is for generating the gentx I propose to explicitly state in the description that 5% is the minimum commission rate. It would help people self debug easier in case the use lower values

Reply:
5% commission is a common practice, it still allow change between 10 to 20 for the bigger companies and looks already tested/approvec

Reply:
Perfect, i will have a PR after this one adding a CI that validates gentx config